### DISCUSSION TITLE
windows11病毒检测

### DISCUSSION THREAD

**Discussion by wl-guan** [bannedbook/fanqiang#879] [Q&A]
win11说
已检测: Trojan:Script/Wacatac.B!ml
状态:已删除
已从此设备中删除威胁或应用。

这是特洛伊木马？为什么有病毒？
![Uploading 1.png…]()

**Comment by bannedbook** [comment]
卸载国产杀毒软件，至少翻墙时不要用它们！因为很多国产杀毒软件，比如360安全卫生、360杀毒软件、腾讯管家、金山卫士等不仅仅会起干扰作用，造成软件无法正常使用或速度变慢，它们与金盾工程有千丝万缕的关系! 可以使用[windows自带的defender杀毒软件](https://docs.microsoft.com/zh-cn/microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-windows?view=o365-worldwide)。

**Reply by wl-guan** [reply]
这个就是windows自带的defender

**Comment by bannedbook** [comment]
看不见图片
哪个文件误报

**Reply by wl-guan** [reply]
已检测: Trojan:Script/Wacatac.B!ml
状态:已删除
已从此设备中删除威胁或应用。
日期: 2024/3/23 9:04
详细信息:
这个程序很危险，而且执行来自攻击者的命令
受影响的项目:
file: C:Downloads\ChromeGo.7z
webfile: C:Downloads ChromeGo.7zhttps://d1.wenxin-ai.top/ChromeGo.7zp

**Comment by bannedbook** [comment]
误报
https://www.virustotal.com/gui/url/6c29dadb719e179adda405c16a933da69b8d97acdb96246f262f3d0f3ffcbd42?nocache=1
